packages/react/test/index.ts

```diff
@@ -35,6 +35,9 @@ reactAxe(React, ReactDOM, 1000, {}, context);
 // axe-core context: string
 reactAxe(React, ReactDOM, 1000, {}, '#container');
 
+// axe-core undefined config
+reactAxe(React, ReactDOM, 1000, {}, undefined)
```

Comment:
The call on line 4 doesn't specify an argument, but the suite doesn't catch a failure.